src/views/mod.rs

```diff
@@ -89,7 +89,11 @@ fn room<'a>(
     let mut yamls = db::get_yamls_for_room_with_author_names(uuid, ctx)?;
     yamls.sort_by(|a, b| a.0.game.cmp(&b.0.game));
     let unique_player_count = yamls.iter().unique_by(|yaml| yaml.0.owner_id).count();
-    let unique_game_count = yamls.iter().unique_by(|yaml| &yaml.0.game).count();
+    let unique_game_count = yamls
+        .iter()
+        .filter(|yaml| !&yaml.0.game.starts_with("Random"))
```

Comment:
You could check for `Random (` instead. It would limit the chance of it breaking randomly.

Reply:
Alternatively, to keep the DB game column clean and slightly easier to search, the count could be removed and done in the room template with my other comment's idea to store the filtered list.